Gruntfile.js

```diff
@@ -0,0 +1,45 @@
+module.exports = function( grunt ) {
+
+	'use strict';
+	var banner = '/**\n * <%= pkg.homepage %>\n * Copyright (c) <%= grunt.template.today("yyyy") %>\n * This file is generated automatically. Do not edit.\n */\n';
+	// Project configuration
+	grunt.initConfig( {
```

Comment:
do we want to consider adding elements of our FE tool chain to fm-zones? Like postcss, jslint, scss linter, imgmin, etc?